akka-http-core/src/main/scala/akka/http/impl/engine/http2/Http2Blueprint.scala

```diff
@@ -158,12 +158,11 @@ private[http] object Http2Blueprint {
           FrameRenderer.render(GoAwayFrame(0, ex.errorCode))
         case ex: StreamTcpException => throw ex // TCP connection is probably broken: just forward exception
         case NonFatal(ex) =>
-          log.error(ex, s"HTTP2 connection failed with error [${ex.getMessage}]. Sending INTERNAL_ERROR and closing connection.")
+          log.debug(s"HTTP2 connection failed with error [${ex.getMessage}]. Sending INTERNAL_ERROR and closing connection.")
```

Comment:
I have to look further into this. Not sure if the previous reporting was really useful.

Reply:
Ok. Strange that this would change with this PR?

Reply:
Indeed, that's not needed yet. I had already added it to make GracfulTerminationSpec tests pass for HTTP/2. I've decided not to add those tests for HTTP/2 right now.

Reply:
I was wrong. Without it Http2PersistentConnectionSpec will now fail.

Reply:
Ok, found the culprit, I also added a cancellation propagation delay to the HTTP2 client side which now changed the error reporting behavior. I'm going to remove that change from this PR.